readme.md

```diff
@@ -84,8 +86,10 @@ This docker-compose file will create a volume called *movienight-config* and aut
 #### Notes for Running Using docker-compose
 The container needs to be restarted to apply any changes you make to *settings.json*.
 
-## Usage
+### FreeNAS - FreeBSD build and run
+An script wich setup an Jail and build and run MovieNight into that Jail as been writen, you'll find it here [freenas-iocage-movienight] (https://github.com/zorglube/freenas-iocage-movienight)
```

Comment:
Remove the space between `]` and `(` so the link renders correctly.

Reply:
Done